server/index.js

```diff
@@ -1,12 +1,40 @@
 const express = require('express');
 const bodyParser = require('body-parser');
+const path = require('path');
 const db = require('../database/index.js');
+const helpers = require('./queryHelpers.js');
 
 const app = express();
 const port = 3002;
 
+// app.use(express.static('../public'));
+app.use(express.static(`${__dirname}/../client/dist/`));
```

Comment:
It sounds like using path.join might be a better idea than writing out the path like this? I'm not super clear on this, though.

Reply:
I fixed this! Kent had the same feedback :) It uses path.join now.